### ISSUE TITLE
syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in helpers line 13.

### ISSUE THREAD

**Issue by ToshKoevoets** [thedevdojo/voyager#2389]
- Laravel Version: 5.4.*
- Voyager Version: 1.0.10
- PHP Version: 7.*

### Description:

syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM)
in helpers line 13.

It's this line, causing this:
return TCG\Voyager\Facades\Voyager::model('Menu')::display($menuName, $type, $options);

Ended up reverting to 1.0.9.

**Comment by emptynick** [comment]
No problem for me.
Did you just call `menu('something')`?

**Comment by ToshKoevoets** [comment]
No. Directly after clean install on existing project.

**Comment by tewfik-dj** [comment]
Same here, just after fresh install a few moments ago
apache2+php 5.6.30-0+laravel 5.4

**Comment by fletch3555** [comment]
yeah, PHP doesn't allow you to call `class`::`method()`::`something` (at least not older versions... PHP 7.* might, but we still support Laravel 5.4, which supports PHP 5.6, so we need to follow that as well), needs to be `class`::`method()`->`something`.  I've found and fixed a couple other instances of that recently.  We'll have to keep a closer eye on that.  In the mean time, would you mind submitting a PR fixing it @emptynick?

**Comment by emptynick** [comment]
@ToshKoevoets are you sure you are using PHP 7.*?
I am, and its not giving me any errors.
@fletch3555 You might just wanna revert [this one](https://github.com/the-control-group/voyager/commit/4e6b7e004e6ea472fb69f015d3e51633cc85484b)

**Comment by sinfuljosh** [comment]
@emptynick that link doesn't need to be fully reverted, just the operator changed on that one line. The other two changes were not breaking anything. 

These 4 changes allowed me to complete a clean install while using Laravel 5.4, Voyager 1.0.10, Php 5.6.32. 

In case anyone has similar issues, I went off of the comment @fletch3555 wrote recently and scope resolution operator (double colon) into an object operator (skinny arrow).  


https://github.com/the-control-group/voyager/blob/v1.0.10/src/Helpers/helpers.php

Line 13 needs to change from: 
`        return TCG\Voyager\Facades\Voyager::model('Menu')::display($menuName, $type, $options);`
to:
`        return TCG\Voyager\Facades\Voyager::model('Menu')->display($menuName, $type, $options);`



https://github.com/the-control-group/voyager/blob/v1.0.10/src/Voyager.php

Line 164 needs to change from:
`            foreach (self::model('Setting')::all() as $setting) {`
to:
`            foreach (self::model('Setting')->all() as $setting) {`

Line 307 needs to change from:
`            $this->permissions = self::model('Permission')::all();`
to:
`            $this->permissions = self::model('Permission')->all();`

Line 322 needs to change from:
`            $this->users[$id] = self::model('User')::find($id);`
to:
`            $this->users[$id] = self::model('User')->find($id);`


I also did a quick grep of all php files looking for ):: and did not find any additional results. So I think those might be the last ones. 
```
[~/voyager/vendor/tcg]# grep --include="*.php" -nRHI ")::" voyager/ 
[~/voyager/vendor/tcg]#
```

**Comment by fletch3555** [comment]
@sinfuljosh thanks for that.  If you'd like to submit the Pull Request for that, I'll gladly accept it

**Comment by sinfuljosh** [comment]
Im never sure I did that right. but there you go.

**Comment by craigweiser** [comment]
When will this fix be available? As far as I can tell, the latest stable release/tag does not yet have this fix.

**Comment by sinfuljosh** [comment]
I see the updated code has been applied to the master branch about 6 hours so it should be live. I don’t have a system I can attempt to create a composer project with php 7.2 at the moment.


But your comment did bring to my attention that the dev branch still is using the incompatible code.

**Comment by emptynick** [comment]
Its is not incompatible, its just PHP >= 7 only.
Maybe this will be a requirement for Voyager 1.1 or 2.

**Comment by fletch3555** [comment]
Guys, I don't tag releases after every single merged pull request.  Please be patient.  I'll likely have a new release out in the next day or two.

As for the 1.1-dev stream, php7 is certainly not a requirement.  I just haven't merged 1.0 into it yet (I usually do that at the same time as when I create releases)
